modules/core/src/main/scala/sbtHood/model/Encoders.scala

```diff
@@ -0,0 +1,11 @@
+package sbtHood.model
+
+import io.circe._
+import io.circe.generic.semiauto._
+
+
+object Encoders {
+
+  implicit val jsonBenchmarkEncoder: Encoder[JsonBenchmark] = deriveEncoder[JsonBenchmark]
+  implicit val jsonPrimaryMetricEncoder: Encoder[JsonPrimaryMetric] = deriveEncoder[JsonPrimaryMetric]
```

Comment:
we are not going to have a lot of encoders/decoders I think we can place all of them on the same file

Reply:
I'd bet for putting them in the companion object of every case class